.github/workflows/linting.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - name: Yamllint
         uses: karancode/yamllint-github-action@master
         with:
+          yamllint_strict: true
```

Comment:
memo) Without this setting, CI will succeed if there are only [warning-level violations](https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration).

https://github.com/ydah/rubocop-rspec/actions/runs/4468847208/jobs/7850130483#step:4:14